CHANGELOG.md

```diff
@@ -117,6 +120,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Removed
 
+- Removed `AverateMethods` enum ([#5657](https://github.com/PyTorchLightning/pytorch-lightning/pull/5689)
```

Comment:
check when `AverateMethods` was added of it was not in any release yet, no need to have this line...